kube/kd/deployment.yaml

```
@@ -205,33 +205,33 @@ spec:
             - name: DB_HOST
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
-                  key: endpoint
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: host
             - name: DB_PORT
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
+                  name: {{.KUBE_NAMESPACE}}-info-rds
                   key: port
             - name: DB_NAME
               valueFrom:
                 secretKeyRef:
-                  name: {{.KUBE_NAMESPACE}}-rds
-                  key: db_name
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: name
             - name: DB_SCHEMA_NAME
               valueFrom:
                 secretKeyRef:
-                  name: hocs-info-service
-                  key: rds_schema
+                  name: {{.KUBE_NAMESPACE}}-info-rds
+                  key: schema
```

`schema` doesn't exist in prod

---

This needs to be `schema_name`.

```suggestion
                  key: schema_name
```